imgui/src/lib.rs

```diff
@@ -551,6 +551,30 @@ impl<'ui> Ui {
     {
         InputInt4::new(self, label, value)
     }
+    #[doc(alias = "InputScalar")]
```

Comment:
Can we add some doc comments here explaining what `InputScalar` is, and maybe in particular how it differs from the other `Input` types (something about basically "let's you do this with u64s and other data types than float or int").